api/spec/requests/spree/api/v2/storefront/products_spec.rb

```diff
@@ -994,6 +994,21 @@
       it_behaves_like 'returns 404 HTTP status'
     end
 
+    context 'with localized_slugs' do
+      let!(:product_slug) { create(:product) }
```

Comment:
I would name this variable `product_with_slugs` (or just `product`) instead of `product_slug`.  `product_slug` implies that the variable stores the slug, not the product itself.